pom.xml

```diff
@@ -25,7 +25,7 @@
   <properties>
     <ome-common.version>6.0.6</ome-common.version>
     <ome-model.version>6.2.2</ome-model.version>
```

Comment:
```suggestion
    <ome-model.version>6.3.1</ome-model.version>
```

Reply:
Sorry, missed this comment, follow PR open now: https://github.com/ome/bio-formats-documentation/pull/256